packages/manager-components/package.json

```diff
@@ -74,7 +74,6 @@
     "react-i18next": "^11.18.1",
     "react-router-dom": "^6.21.0",
     "storybook": "7.5.3",
-    "storybook-addon-react-router-v6": "^2.0.10",
```

Comment:
You sure this doesn't cause any side effect?

Reply:
It is a duplicate line, it already exists at line 40 of the file